### PR TITLE
GitHub health files

### LINKED ISSUE #195: Templates for PRs/Bugs/Features (at request of @BioPhoton)
Posting these at the request of @BioPhoton. 

The following are templates for PRs, feature-requests, and bug-reports I've compiled for the frontend-repo at init.at.  I've tried to filter points from the sources, that apply to that project.

You might wanna remove some points (especially the checklist is rather long and the focus should be on actually using the list / starting slow if necessary, than covering ALL teh things) and go through the sources (see below) again, as I've dropped some points that apply to library development but not app-development. E.g. the most prominent of these is that you'll probably will want bug-reports to include a minimal working example to reproduce the bug.

Any feedback on the templates is very welcome btw (they are somewhat in the need of review) ^^

Gists with the templates:

- [PRs](https://gist.github.com/pheara/d6f946eb5cdeedd0f9197360b0e91e66)
- [Feature requests](https://gist.github.com/pheara/cd795b73028b1269a78eae1fb01e2fef)
- [Bug reports](https://gist.github.com/pheara/8c21b3265c7c80bee9c0b4c07c141101)

Sources:

- https://devchecklists.com/pull-requests-checklist/
- https://www.olioapps.com/blog/pull-request-checklist/
- https://www.pullrequest.com/blog/what-belongs-in-an-effective-code-review-checklist/
- https://medium.com/swift-india/my-pull-request-review-checklist-to-perform-effective-code-reviews-f4bf2b062e6e

For further references/example you might find the list of all `issue_templates.md`s on gh useful: https://github.com/search?utf8=%E2%9C%93&q=in%3Apath+issue_template.md&type=Code&ref=searchresults

### PR DESCRIPTION
**Pull request by BioPhoton**
closes #195 #205

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: "[State]: Title of the bug"
```
Comment (hoebbelsB) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]: What is "[State]"?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+labels: ''
```
Comment (hoebbelsB) [inline .github/ISSUE_TEMPLATE/bug_report.md:5]:
```suggestion
labels: 'state | template'
```
shouldn't we mention to use either state or template label? Wherever it fits best?

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+-->
+
+# Steps to Reproduce the Issue
+<!--
+Explain in detail hoe to reproduce the issue:
+
+1. Do Something
+2. ???
+3. Bug
+
```
Comment (hoebbelsB) [inline .github/ISSUE_TEMPLATE/bug_report.md:33]:
```suggestion
Explain detailed information about the steps how to reproduce this specific bug

```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest a feature for this project
+title: "[State]: Title of the feature request"
+labels: ''
```
Comment (hoebbelsB) [inline .github/ISSUE_TEMPLATE/feature_request.md:5]: same as above

File: .github/SUPPORT.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
+# Support
+
+> This project has a [Code of Conduct][coc].
+> By interacting with this repository, organisation, or community you agree to
+> abide by its terms.
+
+Welcome to RxAngular!  👋
+The toolkit for fully reactive as well as zone-less applications.
```
Comment (hoebbelsB) [inline .github/SUPPORT.md:8]:
```suggestion
The toolkit for fully reactive, high performance angular applications.
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (BioPhoton) [reply .github/ISSUE_TEMPLATE/bug_report.md:4]: Some string helping to categorize the issue. 
We can remove it as we have labels :)

File: .github/SUPPORT.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+We’re excited that you’re using this repository and we’d love to help.
+To help us help you, please read through the following guidelines.
+
+Please understand that people involved with this project often do so for fun,
+next to their day job; you are not entitled to free customer service.
```
Comment (hoebbelsB) [inline .github/SUPPORT.md:13]:
```suggestion
Please understand that people involved with this project often do so in their spare time besides their daily business. You are not entitled to free customer service.
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (BioPhoton) [inline .github/ISSUE_TEMPLATE/bug_report.md:4]:
```suggestion
title: "Title of the bug"
```

Comment (BioPhoton) [inline .github/ISSUE_TEMPLATE/bug_report.md]:
```suggestion
title: "Title of the bug"
```

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
+---
+name: Feature request
+about: Suggest a feature for this project
+title: "[State]: Title of the feature request"
```
Comment (hoebbelsB) [inline .github/ISSUE_TEMPLATE/feature_request.md:4]:
```suggestion
title: "Title of the feature request"
```

**Pull request merged**